### PR TITLE
Fix AutoAWQ -> MLX scales and bias dtype mismatch. Fixes #33.

### LINKED ISSUE #33: fail to run
Image

What I did:

pip install "paroquant[mlx]"

python -m paroquant.cli.chat --model ./Qwen3.5-4B-PARO

macOS  = 15.7.5

### PR DESCRIPTION
**Pull request by jeethu**
`_convert_awq_linear` emitted `fp32` scales and `fp16` biases, but `mx.quantized_matmul` requires matching dtypes for the pair.  This  mismatch produced NaN logits on Apple Silicon, causing generation to stream token 0 ("!") for AutoAWQ paroquant checkpoints.

Text generation works fine after applying this 2 line fix.
image

### CODE REVIEW

**Review by liang2kl — APPROVED** [review]
Good catch. This should be the change in #19 that breaks the inference (as also pointed out in https://github.com/ml-explore/mlx/pull/3434).
